aridanalysis/aridanalysis.py

```diff
@@ -30,11 +42,11 @@ def arid_eda(data_frame, response, features=[]):
     """
     return None
 
-def arid_linreg(data_frame, response, features=[], estimator=None, regularization=None):
+def arid_linreg(df, response, features=[], regularization=None, alpha=1):
```

Comment:
we need to make sure this change is reflected in the other functions (data_frame -> df )

Reply:
Agreed, I like the df usage change though.

Reply:
Ok, we can proceed with that